lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -80,4 +80,8 @@ pysam==0.8.4+gx5
 chronos-python==0.38.0
 
 # GenomeSpace dependencies
-python-genomespaceclient==0.1.8
\ No newline at end of file
+python-genomespaceclient==0.1.8
+
+# OpendID Connect & OAuth2.0
+httplib2==0.10.3
+oauth2client==4.1.2
```

Comment:
What was the rationale for choosing oauth2client over something like PSA? http://python-social-auth.readthedocs.io/en/latest/intro.html PSA gives a huge number of backends implemented for free, that might be a more attractive option than tying us to a custom google oauth2 implementation.